MyLibs/talk2screen.h

```diff
@@ -127,7 +127,7 @@ class ScreenData
 		
 	}
 	/* Initilize screen environment and coordinates */
-	bool initialize(std::vector<std::string> filenames, std::vector<int> patchesOfAreas = {4,4,4});
+	bool initialize(std::vector<std::string> filenames, std::vector<int> patchesOfAreas = {1,4,4});
```

Comment:
We should have one Area. Therefore, patchesOfAreas should be an int number rather than a vector. Correct?

Reply:
That's true, but on this occasion, many variable formats should be modified. For example, vector< string > -- string, vector<vector< int >> -- vector<int>

Reply:
Ok, then keep the data type unchanged, but only put one number in the vector.

Reply:
Got it.